clients/ui/bff/internal/api/middleware.go

```diff
@@ -66,14 +67,23 @@ func (app *App) InjectUserHeaders(next http.Handler) http.Handler {
 	})
 }
 
-func (app *App) enableCORS(next http.Handler) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// TODO(ederign) restrict CORS to a much smaller set of trusted origins.
-		// TODO(ederign) deal with preflight requests
-		w.Header().Set("Access-Control-Allow-Origin", "*")
+func (app *App) EnableCORS(next http.Handler) http.Handler {
+	allowedOrigins, ok := ParseOriginList(app.config.AllowedOrigins)
 
-		next.ServeHTTP(w, r)
+	if !ok {
+		return next
```

Comment:
It's a nitpick and can be done in a FUP PR. Can  we log a warning (error) for invalid entries instead of silently failing?

Reply:
I think maybe my naming isn't so good here - "not ok" in this instance means that the allowlist is empty - as such we don't need a CORS middleware since not having CORS headers is the default - I'll make this more explicit in the follow up.

Reply:
As this is the "normal" use case (yeah, "not ok" being the normal case is weird naming!!) we'd be getting a WARN on probably 99% of deployed use cases.